kiali-server/templates/deployment.yaml

```
@@ -91,6 +89,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: LOG_LEVEL
+          value: {{ include "kiali-server.logLevel" . }}
+        - name: LOG_FORMAT
+          value: {{ .Values.deployment.logger.log_format }}
```

I would also prefer this wrapped in quotes unless there is a reason why it is not. I think we know log_format will always be a string.
```suggestion
          value: "{{ .Values.deployment.logger.log_format }}"
```